lib/mozilla-ast.js

```diff
@@ -1754,8 +1749,9 @@ import { is_basic_identifier_string } from "./parse.js";
     });
 
     def_to_moz(AST_BigInt, M => ({
-        type: "BigIntLiteral",
-        value: M.value
+        type: "Literal",
+        value: M.value,
```

Comment:
```suggestion
        value: typeof BigInt === 'function' ? BigInt(M.value) : null,
```

I happen to know `M.value` is a string, and the spec you linked says it should be a BigInt, so I'm sneaking this fix in!

> In environments that don't support BigInt values, value property will be null as the BigInt value can't be represented natively.